frontend/src/i18n/index.ts

```diff
@@ -16,9 +17,15 @@ i18n
   .init({
     fallbackLng: "en",
     debug: process.env.NODE_ENV === "development",
+    backend: {
+      loadPath: "/locales/{{lng}}/{{ns}}.json",
+    },
+    ns: ["translation"],
+    defaultNS: "translation",
+    supportedLngs: ["en", "zh-CN", "ko-KR", "de"],
```

Comment:
Is this part necessary for some reason? CC @94mashiro who originally did the i18n work

Reply:
sorry for replying late.

the answer is unnecessary, the framework can detect the browser language automatically and find the target translation file(JSON).

all you need is to add the translation content into `frontend/src/i18n/translation.json`, make sure to use the standard language code for the key.

finally, run the `npm run make-i18n` in `/frontend`, or `make build`.

Reply:
@PierrunoYT can you remove this?

Reply:
> @PierrunoYT can you remove this?

Can you do it for me the files are already outdated.

Reply:
> sorry for replying late.
> 
> the answer is unnecessary, the framework can detect the browser language automatically and find the target translation file(JSON).
> 
> all you need is to add the translation content into `frontend/src/i18n/translation.json`, make sure to use the standard language code for the key.
> 
> finally, run the `npm run make-i18n` in `/frontend`, or `make build`.

Can I delete the file?

Reply:
> @PierrunoYT can you remove this?

Can you check again?